src/qt/sendcoinsdialog.cpp

```diff
@@ -735,7 +754,12 @@ void SendCoinsDialog::updateSmartFeeLabel()
 
     int nBlocksToConfirm = (int)25 - (int)std::max(0, std::min(24, ui->sliderSmartFee->value()));
     CFeeRate feeRate = mempool.estimateFee(nBlocksToConfirm);
-    if (feeRate <= CFeeRate(0)) // not enough data => minfee
+    // if SwiftX checked, display it in the label
+    if (ui->checkSwiftTX->isChecked())
+    {
+        ui->labelFeeEstimation->setText(tr("Estimated to get 6 confirmations near instantly with <b>SwiftX</b> !"));
```

Comment:
Nit: inline HTML (though simple bold tag may be ok for transifex), and a space before the `!` that shouldn't be needed.

Reply:
removed the extra space.
I'll remove the bold tag too if it's not ok to have it.